### PR TITLE
Update default behaviours of IOpenApiCustomUIOptions

### PR DESCRIPTION
**Pull request by justinyoo**
Hi, Steve.

I just updated some weird behaviours from your PR branch. Please merge this into your PR branch. Then I'll merge yours to the main branch.

Cheers,

### CODE REVIEW

File: src/Microsoft.Azure.WebJobs.Extensions.OpenApi.Core/Configurations/DefaultOpenApiCustomUIOptions.cs [hunk 1]
```diff
@@ -16,46 +16,62 @@ public class DefaultOpenApiCustomUIOptions : IOpenApiCustomUIOptions
...
 
         /// <inheritdoc/>
-        public virtual async Task<string> GetJavascriptAsync(string filepath = "dist.custom.js")
+        public virtual string CustomStylesheetPath { get; set; } = "dist.custom.css";
+
+        /// <inheritdoc/>
+        public virtual string CustomJavaScriptPath { get; set; } = "dist.custom.js";
+
+        /// <inheritdoc/>
+        public virtual async Task<string> GetStylesheetAsync(string filepath = "dist.custom.css")
```
Comment (ssa3512) [inline src/Microsoft.Azure.WebJobs.Extensions.OpenApi.Core/Configurations/DefaultOpenApiCustomUIOptions.cs:34]: Should we completely drop the file path parameter here and in the interface and just let people override `CustomStylesheetPath`/`CustomJavascriptPath` instead? It seems a bit odd to allow two different potentially conflicting places to override this.

Interface would be
```cs
interface IOpenApiCustomUIOptions
{
        string CustomStylesheetPath { get; } // make read only
        string CustomJavaScriptPath { get; } // make read only
        Task<string> GetJavascriptAsync();
        Task<string> GetStylesheetAsync();
}
```

**Review by ssa3512 — APPROVED** [review]
Looks good to me overall - one comment about the addition of the new `CustomStylesheetPath`/`CustomJavascriptPath` properties duplicating the functionality of `filepath`

File: src/Microsoft.Azure.WebJobs.Extensions.OpenApi.Core/Configurations/DefaultOpenApiCustomUIOptions.cs (hunk 1 above)
Comment (justinyoo) [reply src/Microsoft.Azure.WebJobs.Extensions.OpenApi.Core/Configurations/DefaultOpenApiCustomUIOptions.cs:34]: Oh yes! That would be better!

Comment (justinyoo) [reply src/Microsoft.Azure.WebJobs.Extensions.OpenApi.Core/Configurations/DefaultOpenApiCustomUIOptions.cs]: Updated!

**Pull request merged**